src/mumble/AudioInput.cpp

```diff
@@ -384,6 +384,14 @@ void AudioInput::addMic(const void *data, unsigned int nsamp) {
 					if (qlEchoFrames.isEmpty()) {
 						iJitterSeq = 0;
 						iMinBuffered = 1000;
+					} else if (qlEchoFrames.count() >= 100) {
```

Comment:
Might make sense to introduce a new constant near the sample rate. maybe ```FRAMES_PER_SECOND``` or something like that. That it's 100 is intuitive if you know mumbles insides but otherwise making it descriptive could be quite helpful.